src/lib/y2storage/encryption.rb

```diff
@@ -477,6 +491,13 @@ def supports_label?
       type.is?(:luks2)
     end
 
+    # Whether the attribute #authentication makes sense for this object
+    #
+    # @return [Boolean]
+    def supports_authentication?
+      method.id == :systemd_fde
```

Comment:
Other similar methods use `#is?`.

```suggestion
      method.is?(:systemd_fde)
)```